src/main/java/com/prototype/booking/api/BookMeetingRoomFlow.java

```diff
@@ -0,0 +1,67 @@
+package com.prototype.booking.api;
+
+import com.prototype.booking.api.referencedata.TimingsRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.integration.dsl.IntegrationFlow;
+import org.springframework.integration.dsl.IntegrationFlows;
+import org.springframework.integration.handler.LoggingHandler;
+import org.springframework.integration.http.HttpHeaders;
+import org.springframework.integration.http.dsl.Http;
+import org.springframework.messaging.support.MessageBuilder;
+
+import javax.persistence.EntityManager;
+
+@Configuration
+@Slf4j
+public class BookMeetingRoomFlow {
```

Comment:
```suggestion
public class RoomServiceFlowIntegration {
```

I'm reading DSL overall as GatewayIntegration & FlowIntegration

Reply:
Yes